main.py

```
@@ -1,44 +1,437 @@
+                "Test siqnalı analitik tərəfindən hazırlanır.\n\n"
+                "📲 Proses: Əvvəlcə 'Hazırlıq' bildirişi alacaqsınız.\n"
+                "Bildirişdən 10-15 saniyə sonra siqnal göndəriləcək.\n\n"
+                "Xahiş olunur gözləmədə qalın.",
+                uid, call.message.message_id
+            )
+        except Exception as e: print(e)
+
+    elif call.data == "show_rules":
+        # YOXLA: TEST ETMƏYİB VIP OLA BİLMƏZ
+        if not test_used and user_status not in ['test_sent', 'vip', 'awaiting_proof'] and int(uid)!= ADMIN_ID:
+            bot.answer_callback_query(call.id, "❌ Əvvəlcə test siqnalını yoxlamalısınız", show_alert=True)
```

Ruff (0.15.12)

[warning] String contains ambiguous `ı` (LATIN SMALL LETTER DOTLESS I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)